webpack.config.js

```diff
@@ -736,6 +736,7 @@ module.exports = (env, argv) => {
             new CopyWebpackPlugin({
                 patterns: [
                     "res/apple-app-site-association",
+                    "res/.well-known/assetlinks.json",
```

Comment:
No sure that the file will be copied to the correct destination. Can you confirm @t3chguy ?

Sorry I am a newbie here :)

Reply:
Yes, it was not working as expected.
After [095c613](https://github.com/element-hq/element-web/pull/27439/commits/095c613b6980a5d425d48e7ddabfb6204e17f57e) it's fine.

Tested OK using `./scripts/ci_package.sh`